spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/pom.xml

```diff
@@ -14,6 +14,10 @@
         <version>1.2.0.BUILD-SNAPSHOT</version>
     </parent>
 
+    <properties>
+        <grpc-google-cloud-trace.version>0.69.0</grpc-google-cloud-trace.version>
```

Comment:
Hmm, so unless you do this, everything breaks? How will our users know to do this downgrade? Can we just exclude the transitive import from `zipking-gcp` dependency?

Reply:
This is just a test dependency, so we can query Stackdriver (v2 does not have get/list yet). But actually, the version is not necessary at all -- it's brought in through cloud BOM by way of `spring-cloud-gcp-dependencies`. 
I'll remove the property.